applications/dotwise.md

```diff
@@ -0,0 +1,191 @@
+
+In addition to my technical skills, I have also been actively involved in the wider tech community. I have served as a Chainlink Community Advocate, Aave Turkey Community Manager, and Founding Chair of Gazi University ACM Student Chapter. My previous role as a Microsoft Learn Student Ambassador has also given me the opportunity to share my knowledge and experience with others. I have set of experiences and skills and particularly in the areas of full stack software development and community management.
+
+Also I developed Awesome Polka for W3F and now working on user base and marketing.
+
+### Team Code Repos
+
+- https://github.com/tolgayayci
+
+### Team LinkedIn Profiles (if available)
+
+- https://www.linkedin.com/in/tolgayayci
+  
+## Development Status :open_book:
+
+Some of the mockups and tech stacks has been prepared as mentioned above. I made detailed research about and waiting to start project as soon as possible. There isn't any available codebase right now.
+
+## Development Roadmap :nut_and_bolt:
+
+### Overview
+
+This RFP is planned as 2 milestones, it will be completed in three months.
+
+- **Total Estimated Duration:** ~3 months
+- **Full-Time Equivalent (FTE):**  1 FTE
+- **Total Costs:** $30,000
+
+### Milestone 1 — Backend
+
+- **Estimated duration:** 1,5 months
+- **FTE:**  1
+- **Costs:** 15,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | MIT |
+| **0b.** | Documentation | I will provide both **inline documentation** of the code and a basic **tutorial** that explains how backend architecture works.|
+| **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| **0d.** | Docker | I will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone. |
+| 1. | SubQuery Integration | I will integrate SubQuery, into the backend of the project. This will allow users to search, filter, and aggregate data in a more efficient manner. Users just need to select attributes on UI to write a query with SubQuery via Core API. |
+| 2. | Database Integration | I will integrate MongoDB to store queries, visualizations, some stats and so more. |
+| 4. | Core API | The platform will offer an API that allows to gather, retrieve, and update project-related data. The use of FastAPI will help to ensure high performance and easy integration with other components. |
+| 5. | Analytics | I will integrate monitoring solutions to forecast backend system like Grafana. |
```

Comment:
Of course, that means analyses of the backend system with grafana. This could be Core API analysis, Subquery usage analysis and so more. For example, Grafana firstly will be used to monitor subquery usage, if I monitor the system well, I can make my improvements better.